### ISSUE TITLE
Help with diagnosing abort on save

### ISSUE THREAD

**Issue by ItsMeOnly2** [Beep6581/RawTherapee#5650]
I'm fairly sure the problem lies with my setup, but I truly hope you can help me out with rooting out the issue:

I'm running a basically do-it-yourself linux system (uname -a: Linux beethoven 3.18.46 #4 SMP PREEMPT Wed Dec 28 19:36:22 CET 2016 x86_64 GNU/Linux) based on Debian base and Debian-maintained glibc, gcc and binutils. Basically, I can save or export image to gimp as long as I don't change any parameter. If I do, rawtherapee (any version/build) aborts with message "corrupted size vs. prev_size" upon trying to save.
Attaching gdb output
[log.txt](https://github.com/Beep6581/RawTherapee/files/4188136/log.txt)
and strace
[rawtherapee.log](https://github.com/Beep6581/RawTherapee/files/4188137/rawtherapee.log)

Any help would be appreciated.

**Comment by Beep6581** [comment]
Apparently a result of a memory leak. Since no one else reported this, and since you're using a self-rolled distro, I'd assume it's related to using too-old/too-new components in your toolchain. A complete build log would give more clues - including all cmake arguments, dependency versions, and anything else related.

**Comment by ItsMeOnly2** [comment]
Attaching:
[CMakeOutput.log](https://github.com/Beep6581/RawTherapee/files/4188807/CMakeOutput.log)
[CMakeError.log](https://github.com/Beep6581/RawTherapee/files/4188808/CMakeError.log)
[CMakeCache.txt](https://github.com/Beep6581/RawTherapee/files/4188809/CMakeCache.txt)
Anything else I can give you that would give you a clue?

**Comment by ItsMeOnly2** [comment]
Just to add to the weirdness: I started using AppImage you provided and while it doesn't crash when an edit is made and then saved... it does when an edit is made aftewards (i.e. on second save) with munmap_chunk(): invalid pointer.

Recently I completely overhauled my stuff and using most recent everything, so I'm completely at loss what is going on. I recompiled RT yesterday and converted several files without issue, only to have it crash on first file today - same thing.
